exercises/rust-2-a/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+pub struct Hand {
```

Comment:
The second part of the exercises are structured a bit differently. There should be multiple starting points, which are defined in the exercise's `.meta/start/<start number>` directory. For an example see the Go 2-a exercise's `start` directory: https://github.com/exercism/research_experiment_1/tree/master/exercises/go-2-a/.meta/start

The idea is that in the second part of the exercise, students get assigned one of the different starting points (which should use different approaches), and then expand/modify the code to fit the updated requirements. That way, we can see how the different starting points influence the outcome. Sorry that this wasn't clear enough.